src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square," + " area: "
+                + getArea() + " sq. units, side: "
+                + side + " units, color: " + super.getColor());
```

Comment:
The usage of 'super.getColor()' is correct, but ensure that the 'getColor' method in the parent class 'Figure' is not static. Static methods in inheritance hierarchies can lead to brittle code.